packages/core/src/routes/interaction/additional.test.ts

```diff
@@ -0,0 +1,199 @@
+import { ConnectorType } from '@logto/connector-kit';
+import { demoAppApplicationId, InteractionEvent } from '@logto/schemas';
+import { createMockUtils } from '@logto/shared/esm';
+
+import { mockSignInExperience } from '#src/__mocks__/sign-in-experience.js';
+import RequestError from '#src/errors/RequestError/index.js';
+import type koaAuditLog from '#src/middleware/koa-audit-log.js';
+import { createMockLogContext } from '#src/test-utils/koa-audit-log.js';
+import { createMockProvider } from '#src/test-utils/oidc-provider.js';
+import { MockTenant } from '#src/test-utils/tenant.js';
+import type { LogtoConnector } from '#src/utils/connectors/types.js';
+import { createRequester } from '#src/utils/test-utils.js';
+
+import { verificationPath, interactionPrefix } from './const.js';
+
+const { jest } = import.meta;
+const { mockEsmWithActual } = createMockUtils(jest);
+
+// FIXME @Darcy: no more `enabled` for `connectors` table
```

Comment:
@darcyYe please confirm and maybe an issue tracking this?

Reply:
LOG-7148